src/modules/InsightsLibrary/components/ViewEdit/ModalSettings/presenter.jsx

```diff
@@ -42,7 +43,8 @@ function ModalAccessSettings(props) {
           <div {...classes('setting')}>
             <div {...classes('setting-title')}>Publish state</div>
             <Tabs
-              {...classes('setting-value')}
+              {...classes({element: 'setting-value',
```

Comment:
Don't forget about formatting (spaces near curly brackets)

Reply:
this is actual, when we do import or parameters pass. here everything is ok

Reply:
try to find 'classes({ ' across the project, and after try 'classes\(\{\S'